lab-raziyeh/app/component/navbar/_navbar.scss

```diff
@@ -0,0 +1,21 @@
+@import  "_theme";
+
+.landing {
+  background-color: $app-secondary;
+}
+
+.navbar-inverse {
+  background-color: #6f94a2 !important;
```

Comment:
@Raziyehbazargan `!important` flags are practically **never** needed - unless there is a super edge case where something has to be overwritten.  All of these should be removed.